azurerm/data_source_api_management_api.go

```diff
@@ -0,0 +1,192 @@
+package azurerm
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2018-01-01/apimanagement"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceApiManagementApi() *schema.Resource {
```

Comment:
For `acronyms` like `API` (application programming interface ), according to [Golang Code Review Guideline](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), it recommends either `API` or `api`.

Can we follow up this guideline ?

Reply:
possibly in the future - but we're intentionally not following this (across providers) at this time (e.g. Arm/Api/Dns/MySql/Sql) since it impedes readability